src/pybind/mgr/dashboard/frontend/src/app/shared/components/components.module.ts

```diff
@@ -36,7 +36,12 @@ import {
   SelectModule,
   ComboBoxModule,
   ProgressIndicatorModule,
-  InlineLoadingModule
+  PanelModule,
+  LayoutModule,
+  TilesModule,
+  PopoverModule,
+  InlineLoadingModule,
```

Comment:
don't import all these modules. you only need the `TagModule` so just resort to importing only that. the other changes are different.
```suggestion
  InlineLoadingModule,
```